stdlib/types/ints.nix

```
@@ -0,0 +1,31 @@
+{ ... }:
+{
+  isType = builtins.isInt;
+
+  even = a: (mod a 2) == 0;
+  odd = a: (mode a 2) != 0;
```

to be consistent with `isType`:

```suggestion
  isEven = a: (mod a 2) == 0;
  isOdd = a: (mode a 2) != 0;
```